docs/data-sources/iam_policy.md

```diff
@@ -10,14 +10,19 @@ description: |-
 Generates an IAM policy that may be referenced by and applied to other HCP IAM resources, such as the `hcp_project_iam_policy` resource.
 
 To see what each role grants, please see [HCP
-Documentation](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#organization-role).
+Documentation](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#organization).
 The basic roles can be referenced as follows:
 
 * `roles/owner`
 * `roles/admin`
 * `roles/contributor`
 * `roles/viewer`
 
+You can now assign HCP Vault Secrets specific roles, please see [Documentation](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#project)
```

Comment:
I personally would prefer we remove the "You can now" verbiage. Who is going to come clean that up when it has been available for months?